### PR TITLE
move apis into ProfileView widget and create Timeline Widget

### PR DESCRIPTION
**Pull request by elenagiere**
now that timeline doesn't have api calls from within, we need to move them up to the widget level.

### CODE REVIEW

**Review by venkateshwarv — CHANGES_REQUESTED** [review]
Looks good otherwise.

File: ProfileView/src/[sandbox]/sandbox.ts [hunk 1]
```diff
@@ -93,6 +93,9 @@ export class Sandbox extends LitElement {
               customer="560021-Venki"
               .template=${sampleTemplate}
               .authToken=${"st=demoassure&so=sandbox&ss=stream&sp=w&se=2021-04-06T07:38:17Z&sk=sandbox&sig=qnKHkG1aAZryxbBfgTLG1XR8jLFbztQ4xKyn5APjdSY="}
+              type="journey-and-stream"
+              stream-id="st=demostore&so=sandbox&ss=datasink&sp=w&se=2021-04-23T18:25:43.511Z&sk=sandbox&sig=TEUtd3qKp6pYjoTM7GEHDZeKRfnIWr90MQoW6r2xsB0="
```
[the remark this answers is not in the document]
Comment (elenagiere) [reply ProfileView/src/[sandbox]/sandbox.ts:97]: do you want to just have authToken then? and then reuse that single property?

[the remark this answers is not in the document]
Comment (elenagiere) [reply ProfileView/src/[sandbox]/sandbox.ts:97]: I don't understand what you are saying. do you want to keep both as separate properties? or not?

File: ProfileView/src/[sandbox]/sandbox.ts [hunk 2]
```diff
@@ -93,6 +93,9 @@ export class Sandbox extends LitElement {
...
+              pagination="$top=15"
```
[the remark this answers is not in the document]
Comment (elenagiere) [reply ProfileView/src/[sandbox]/sandbox.ts:98]: sounds good. I will do this when defining the ProfileView Property pagination
`  @property({ reflect: true }) pagination: string = "$top=15";`

**Review by futurethang — APPROVED** [review]

File: ProfileView/src/[sandbox]/sandbox.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (elenagiere) [reply ProfileView/src/[sandbox]/sandbox.ts]: I ended up having just authToken Property for both.

**Pull request merged**